frontend/lib/types/Param.cpp

```diff
@@ -946,6 +946,11 @@ IMPLEMENT_DUMP(Param);
 // clear the macros
 #undef PARAM_NODE
 
+const EnumParam* Param::getEnumParam(Context* context, ID id) {
+  auto ast = parsing::idToAst(context, id)->toEnumElement();
+  auto value = std::make_pair(id, ast->name().str());
+  return EnumParam::get(context, value);
```

Comment:
I can't think of a way to "disable" this one since you use it in your implementation of `getEnumParam`. C clang-based linter would be the only solution I can think of, but that's involved and pretty slow.